File: JS Fundamentals/JS_Operators_and_Expressions_HW/JS_Operators_and_Expressions_HW/Task_03/Rectangle_Area.js
/// <reference path="C:\Users\M.Tonkov\Documents\Visual Studio 2013\Projects\JS_Operators_and_Expressions_HW\JS_Operators_and_Expressions_HW\js-console.js" />

//Write an expression that calculates rectangle’s area by given width and height.

function area() {
    var width = document.getElementById("user-input-width").value,
        height = document.getElementById("user-input-height").value;

    if (!isNaN(height) && !isNaN(width)) {
        jsConsole.writeLine("Rectangle's area is: " + (width * height));
    } else {
        jsConsole.writeLine("Please enter valid numbers for width and height");
    }
}

File: CSharp 1/HW.OperatorsExpressionsStatements/Hw03_RectangleArea/RectangleArea.cs
using System;


namespace Hw03RectangleArea
{
    class RectangleArea
    {
        static void Main()
        {
            //Write an expression that calculates rectangle’s area by given width and height.

            Console.WriteLine("Please enter rectangle's width: ");
            double width = double.Parse(Console.ReadLine());

            Console.WriteLine("Please enter rectangle's height: ");
            double height = double.Parse(Console.ReadLine());

            double rectangleArea = height * width;

            Console.WriteLine("Rectangle's area is: {0:0.00}", rectangleArea);

        }
    }
}
